server/src/index.ts

```diff
@@ -4,10 +4,11 @@ import 'geofire-common'
 import { Message } from './types/Message';
 import { createMessage } from './actions/createMessage'
 import { createUser } from './actions/createConnectedUser'
-import { toggleUserConnectionStatus, updateUserLocation } from './actions/updateConnectedUser'
+import { toggleUserConnectionStatus, updateUserLocation, updateUserDisplayName } from './actions/updateConnectedUser'
 import { deleteConnectedUserByUID } from './actions/deleteConnectedUser'
+import { getConnectedUserDisplayName } from './actions/getSingleConnectedUser'
 import {geohashForLocation} from 'geofire-common';
-import { findNearbyUsers } from './actions/getConnectedUsers'
+import { findNearbyUsers } from './actions/getMultipleConnectedUsers'
```

Comment:
Keep everything within getConnectedUsers. We should try and get relevant functions within the same files.

Get MultipleConnectedUsers should change back to getConnectedUsers!